src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = (value) => {
+    if (value === undefined) {
+      const count = amount;
+
+      amount = 0;
+
+      return count;
+    } else {
+      amount += value;
```

Comment:
I hear the mentor's voice here: 'value is a bad name, son'... 'may the force be with you' 😎

Reply:
fixed )